notebooks/week3/04.AB_test_model_serving.py

```
@@ -0,0 +1,360 @@
+        pass
+    else:
+        raise e
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ### Call the endpoint
+
+# COMMAND ----------
+
+token = dbutils.notebook.entry_point.getDbutils().notebook().getContext().apiToken().get()
```

Ruff

Undefined name `dbutils`

(F821)